src/tools/r2rdump/R2RMethod.cs

```diff
@@ -117,6 +65,11 @@ class R2RMethod
         /// </summary>
         public SignatureType[] ArgTypes { get; }
 
+        /// <summary>
+        /// The type that the method belongs to
+        /// </summary>
+        public string DeclaringType { get; }
```

Comment:
Is the main purpose of this to get a more qualified type name? If so, I think it would be even better if we could get the fully qualified which would include the namespace.

Reply:
Yes, this gets the class that the method belongs to. My new commit would get the full namespace